src/controllers/Lane.js

```diff
@@ -170,9 +175,12 @@ class Lane extends Component {
           className="react-trello-card"
           onDelete={onDeleteCard}
           onClick={e => this.handleCardClick(e, card)}
+          onChange={updatedCard => this.updateCard(updatedCard)}
           showDeleteButton={!hideCardDeleteIcon}
           tagStyle={tagStyle}
           cardDraggable={cardDraggable}
+          editable={editable}
+          t={t}
           {...card}
```

Comment:
Pls make sure `onCardUpdate` is not passed to `Section` styled component to avoid the warning on the console. One way is to whitelist it in the list of props to exclude it from being added to `otherProps`